src/pulselistener/pulselistener/mercurial.py

```diff
@@ -45,6 +45,7 @@ def __init__(self, config, cache_root):
         self.try_url = config['try_url']
         self.try_mode = TryMode(config.get('try_mode', 'json'))
         self.try_syntax = config.get('try_syntax')
+        self.try_name = config.get('try_name', 'try')
```

Comment:
Could this be generated from try_url, so we don't need another config?

Reply:
Hardly, or in a weird way:

- for `try`, the path in the url is `/try`
- for `nss-try` the path in the url is `/projects/nss-try`

We would assume to skip things, etc...
There may be a better way with the treeherder API ?

Reply:
It looks like the repositories supported by TreeHerder are hardcoded in this file: https://github.com/mozilla/treeherder/blob/master/treeherder/model/fixtures/repository.json.
Maybe you could GET it (well, the raw version) and use the info from there.

Reply:
Are you kidding me ? This is awesome !

Reply:
https://github.com/mozilla/release-services/issues/2183